### PR TITLE
feat(tutorial): complete step 2

### PR DESCRIPTION
**Pull request by lucaszorzo**
Closes #

{{short description}}

#### Changelog

**New**

- {{new thing}}

**Changed**

- {{change thing}}

**Removed**

- {{removed thing}}

### CODE REVIEW

File: src/content/LandingPage/LandingPage.js [hunk 1]
```diff
@@ -1,7 +1,100 @@
...
+  tab: {
+    href: '#',
+    role: 'presentation',
+    tabIndex: 0,
+  },
+};
 
 const LandingPage = () => {
-  return <div>LANDING PAGE</div>;
+  return (
```
Comment (kellychurchill) [inline src/content/LandingPage/LandingPage.js]: You're missing this div wrapper: `<div className="bx--grid bx--grid--full-width landing-page">`

**Comment by kellychurchill** [comment]
Your preview build https://deploy-preview-2145--carbon-tutorial.netlify.com/#/ does not look like the expected output: https://www.carbondesignsystem.com/tutorial/react-step-2

Your middle row should have a white background. Please try again and be sure to review your changes locally before pushing updates to this PR.

**Comment by lucaszorzo** [comment]
I'm missing what's causing the right tabs to not be included on grey area
help ?
image

**Comment by kellychurchill** [comment]
Hmm hard to say without seeing your code. Feel free to push to this PR and I will review it.

**Pull request closed without merging**